### ISSUE TITLE
Error: FxA returned requested scopes are not allowed.

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#7277]
Sentry Issue: [GUARDIAN-4T](https://mozilla.sentry.io/issues/3912905453/?referrer=jira_integration)

```
Error: FxA returned requested scopes are not allowed.
  File "/home/node/guardian-website/dist/lib/fxa.js", line 215, in Object.fetchSubscriptionBillingDetails
    throw new FxaScopesNotAllowedError();
  File "/home/node/guardian-website/dist/controllers/api/subscription-details.js", line 33, in subscriptionDetails
    const fxaSubscriptionBillingDetailsData = await FXA.fetchSubscriptionBillingDetails(accessToken);
```

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-5061)
┆Reporter: Jeremy Swinarton

**Comment by data-sync-user** [comment]
➤ Jeremy Swinarton commented:

PR https://github.com/mozilla-services/guardian-website/pull/1661 ( https://github.com/mozilla-services/guardian-website/pull/1661|smart-link )  was merged and released as part of 1.31 but this does not seem to have resolved the issue – reopening.

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

Note to self: Kept “High” as it’s seen as necessary for the non-IAP upgrades epic.

**Comment by data-sync-user** [comment]
➤ Jeremy Swinarton commented:

We attempted a second fix for this in release 1.32 but were still not able to get to the root of the issue so this has been re-opened.
